### DISCUSSION TITLE
FA : taxes on buy/sell operations

### DISCUSSION THREAD

**Discussion by sergeybykov85** [aptos-labs/aptos-developer-discussions#555] [Questions] [answered]
Hi everybody.

Please advise if somebody know the answers. 

I want to create fungible token that follows the exactly Fungible Asset (FA) standard (not the Coin legacy)
and I try to fingure out is there any DEX platform supported FA standard?
I see that popular Liquidswap works only with Coins, AptoSwap also works only with Coins as well.
Does anybody know any DEX that supports exactly FA standard (or both FA and Coin standard)?

I want to override deposit/withdraw methods in my FA token and apply Taxes for Sell/Buy operations.
I did such things in solidity smart contract and integrated with pancake/uniswap dex.
I want to do the same experiments with Aptos now.
But no luck to find the Dex, please advise.
If somebody already did the same exercise (taxes for buy/sell) and can share the sample, then it would be great as well.

BTW, Why liquidswap supports only Coins standard?

Thanks

**Comment by Alexzander-Stone** [comment, marked as answer]
Hey there @sergeybykov85, [ThalaSwap](https://twitter.com/ThalaLabs/status/1866477497627324686) supports the FA standard!
